server/routes/all.ts

```diff
@@ -24,6 +26,16 @@ export default function routes(userService: UserService): Router {
     authorisationMiddleware([manageIncentiveLevelsRole]),
     incentiveLevelRoutes(standardRouter(userService)),
   )
+  router.use(
+    '/incentive-reviews/prisoner',
+    authorisationMiddleware(['ROLE_MAINTAIN_IEP']),
```

Comment:
does a user require the `ROLE_MAINTAIN_IEP` role just to see incentive reviews? i think it's only needed to update them so this and line 36 should go.

Reply:
error, line 36 should remain